Sources/StreamChatSwiftUI/ChatChannel/MessageList/AsyncVoiceMessages/VoiceRecordingContainerView.swift

```diff
@@ -88,7 +89,6 @@ public struct VoiceRecordingContainerView<Factory: ViewFactory>: View {
             player.subscribe(handler)
         }
         .padding(.all, 8)
-        .background(Color(colors.background))
```

Comment:
Text colour change is good, but removing the background would diverge from the default design which we can't make. Could you please exclude the background colour change and then we are good to take it in.
Thank you for contributing!

Reply:
Without this change, it looks like my first picture, this is a bug. I removed this color because I don't think this container view should have a bg of its own, since it should just rely on a message bubble to provide the color for the both of them, no? Using a hardcoded color here is wrong however you look at it - sender and receiver might have different bg colors. In any case, if you don't like the way I fixed it, please fix it however you see fit, but right now it looks wrong and needs to be fixed.

Reply:
Hi @f3dm76, we have opened a [PR:689](https://github.com/GetStream/stream-chat-swiftui/pull/689) for fixing the issue with the layout highlighted here.

Reply:
Related PR has been merged

Reply:
hope that one includes text colors too, so I'm closing this one, thank you